### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by arya1706**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ A short and simple program to help others learn basic of software development.
 Members of LLT
 - Fahad
 - Ameen
+- Arya
```
Comment (Fadeeii) [inline README.md:12]: - Fadeela

**Comment by arya1706** [comment]
arya

**Comment by Ananya-1219** [comment]
Ananya

**Comment by Ranger-NF** [comment]
Everyone should create their own repo (by forking) and make changes in that.
@Ananya-1219 @Fadeeii 

This one is made by @arya1706

**Pull request merged**